gui/src/calibrateble.cpp

```diff
@@ -159,6 +162,16 @@ void CalibrateBLE::showEvent(QShowEvent *event)
 {
     Q_UNUSED(event);
     ui->chkUseMagnetometer->setChecked(!trkset->getDisMag());
+    if(trkset->hardware().contains("XIAO")) {
```

Comment:
Will fix this in the hw_feat branch as well as some of the other board specific items.